application-helm/templates/create-mysql-app-db-user.yaml

```diff
@@ -0,0 +1,84 @@
+{{- if .Values.application.persistance -}}
+{{- if .Values.application.persistance.mysql.auto_db_user_create -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: createdb-
```

Comment:
this can be 
```
{{ template "application.name" $ }}-createdb-
```
so that we know job ran for which app

Reply:
Good point

Reply:
Seems we already have persistence added to the values.yaml

```
persistence:
# -- Add Storage volumes(PVCs) to the pods
  enabled: false
  mountPVC: false
  mountPath: "/"
  accessMode: ReadWriteOnce
  ## If defined, storageClass: <storageClass>
  ## If set to "-", storageClass: "", which disables dynamic provisioning
  ## If undefined (the default) or set to null, no storageClass spec is
  ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
  ##   GKE, AWS & OpenStack)
  ##
  storageClass: ""
  # Additional label added on PVC 
  labels:
    # key: "value"
  annotations:
  #  "helm.sh/resource-policy": keep
  storageSize: 8Gi
```